lib/server-secrets.js

```diff
@@ -1,22 +1,15 @@
 'use strict'
 
-// Everything that cannot be checked in but is useful server-side is stored in
-// a JSON data file: private/secret.json.
-
 const fs = require('fs')
 const path = require('path')
-
-const secretsPath = path.join(__dirname, '..', 'private', 'secret.json')
-
-if (fs.existsSync(secretsPath)) {
-  try {
-    module.exports = require(secretsPath)
-  } catch (e) {
-    console.error(`Error loading secret data: ${e.message}`)
-    process.exit(1)
-  }
-} else {
-  console.warn(
-    `No secret data found at ${secretsPath} (see lib/server-secrets.js)`
+const legacySecretsPath = path.join(__dirname, '..', 'private', 'secret.json')
+if (fs.existsSync(legacySecretsPath)) {
```

Comment:
Is there a reasonable possibility that a self-hoster could have moved to the new model (added env vars and/or yml config file) already but left `private/secret.json` behind? If so, maybe the message could include something about detecting `private/secret.json` and that the user needs to delete it?

Reply:
Want to go ahead and edit it?

Reply:
Will do